### PR TITLE
[Release] Merge release branch 1.9.0 back to main

### PR DESCRIPTION
**Pull request by 0mza987**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: examples/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-promptflow[azure]
+--extra-index-url https://azuremlsdktestpypi.azureedge.net/promptflow/
```
Comment (zhengfeiwang) [inline examples/requirements.txt:1]: merge blocker/reminder: manually bump versions in pyproject.toml to 1.10.0.dev0

**Review by wangchao1230 — DISMISSED** [review]

File: examples/requirements.txt (hunk 1 above)
Comment (0mza987) [reply examples/requirements.txt]: Updated.

**Review by crazygao — DISMISSED** [review]

**Review by brynn-code — DISMISSED** [review]

**Review by wangchao1230 — APPROVED** [review]

File: src/promptflow-azure/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # Release History
 
-## 1.9.0 (Upcoming)
+## 1.9.0 (2024.04.17)
```
Comment (wangchao1230) [inline src/promptflow-azure/CHANGELOG.md:3]: add a new 1.10 entry with Upcoming so devs know where to put new changelog

Comment (0mza987) [reply src/promptflow-azure/CHANGELOG.md:5]: Updated

**Review by lalala123123 — APPROVED** [review]

**Review by zzn2 — APPROVED** [review]

**Pull request merged**